roomsrv/server.go

```diff
@@ -90,6 +92,7 @@ func New(
 	s.authorizer = membersdb
 
 	s.Members = membersdb
```

Comment:
@cryptix what's the point of having both s.authorizer & s.Members?

Reply:
residual cruft, really. it can be removed now.